### DISCUSSION TITLE
Need a maintainer 🧑‍💻🧑‍💻

### DISCUSSION THREAD

**Discussion by KoditkarVedant** [notion-dotnet/notion-sdk-net#349] [Announcements]
I try my best to keep the SDK up to date but for past month or so unable to give enough attention to the project. I was thinking if anybody would like to be a maintainer. So, It won't be solely depend on me and the library will continue to be up to date with the APIs.

Do let me know below in thread if you are interested.

🤗 Don't worry I'm not abandoning the project - I'll be slow to respond to your queries.

Always welcome to open issue or PR - would be happy to accept.

Once again thanks for the support we are getting for this library we already have 92 stars. Looking forward to soon cross the  💯 mark. 

🎉🎉🎉

**Comment by alejandrogsanz** [comment]
Hi, I'm interested as I'm using the library quite a lot for a personal project and I've seen that there are some bugs that prevent me from performing certain actions. Although I have downloaded the code and created a custom version, it would be nice to be able to integrate it into the library officially so that everyone can use it correctly. Also, i would update changes from the Notion API as soon as i can.

**Reply by KoditkarVedant** [reply]
That would be great. 👏🏼  Feel free to open an issue and pull request for bug or changes you would like.

**Comment by bstag** [comment]
Can we get some access for those that want to continue with this to allow us/them to pull recent requests and push out releasees?
